Cartfile

```diff
@@ -1,3 +1,5 @@
 github "Alamofire/Alamofire"
 github "Alamofire/AlamofireImage"
 github "SwiftyJSON/SwiftyJSON"
+github "AliSoftware/OHHTTPStubs" "swift-3.0"
```

Comment:
https://github.com/AliSoftware/OHHTTPStubs#using-the-right-swift-version-of-ohhttpstubs-for-your-project

> The project is currently set up with SWIFT_VERSION=2.3 on master.
This means that the framework on master will build using Swift 2.2 on Xcode 7 and Swift 2.3 on Xcode 8.
If you want Carthage to build the framework with Swift 3.0, you can use the swift-3.0 branch, whose only difference with master is that the project's Build Settings set SWIFT_VERSION=3.0 instead of 2.3.
Note: Later, probably a few weeks after the Xcode 8 official release, we plan to merge the swift-3.0 branch into master and create a compatibility branch swift-2.3 for Carthage users who have not migrated their Swift code yet. We'll try to keep the branch up-to-date with master as best we can.
Hopefully, Carthage will address this in a future release. See the related issue Carthage/Carthage#1445.

現時点ではswift-3.0ブランチを使う必要があるんですね、了解です
（折りたたまれてて最初わからんかった……）